packages/ui/src/models/camel/camel-route-resource.ts

```diff
@@ -53,7 +54,9 @@ export class CamelRouteResource implements CamelResource, BeansAwareResource {
 
   getVisualEntities(): CamelRouteVisualEntity[] {
     return this.entities.filter(
-      (entity) => entity instanceof CamelRouteVisualEntity || entity instanceof CamelOnExceptionVisualEntity,
+      (entity) =>
+        entity instanceof CamelRouteVisualEntity ||
```

Comment:
In the future, the idea is to remove this section by moving `CamelRouteVisualEntity` to the `SUPPORTED_ENTITIES` array, this way, we just need to update the `SUPPORTED_ENTITIES` array whenever there's a new Visual entity to support.